docker/1.4.0/final/Dockerfile.eia

```diff
@@ -0,0 +1,41 @@
+ARG py_version
+
+# TODO: the base image name may change
+FROM awsdeeplearningteam/mxnet-model-server:base-cpu-py$py_version
+
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends \
+    libopencv-dev \
+    build-essential \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /
+
+# Will install from pypi once packages are released there. For now, copy from local file system.
+# TODO: The name for the tar.gz binary may change
+COPY dist/sagemaker_mxnet_container-2.0.0.tar.gz /sagemaker_mxnet_container-2.0.0.tar.gz
+
+RUN pip install --no-cache /sagemaker_mxnet_container-2.0.0.tar.gz && \
+    \
+    rm /sagemaker_mxnet_container-2.0.0.tar.gz
+
+RUN pip install --no-cache onnx==1.4.1 \
+                           keras-mxnet==2.2.4 \
+                           https://s3.amazonaws.com/amazonei-apachemxnet/amazonei_mxnet-1.4.0-py2.py3-none-manylinux1_x86_64.whl
```

Comment:
I think the guidance before was to do all the pip installs in one go to make the image smaller

Reply:
merged